### PR TITLE
Rockfield: Add color annotations support

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

This PR is built on top of https://github.com/Automattic/themes/pull/3223 and adds support for color annotations on Rockfield using the CSS variable approach that is already included on Hever.

Screenshot 2021-02-10 at 12 43 49

Screenshot 2021-02-10 at 12 43 56

### CODE REVIEW

File: varia/inc/wpcom-colors-v2.php [hunk 1]
```diff
@@ -0,0 +1,212 @@
+<?php
+// Custom Colors:
+function varia_define_color_annotations( $colors ) {
+	// Background Color
+	if ( isset( $colors['background'] ) ) {
+		// --wp--preset--color--background
```
Comment (MaggieCabrera) [inline varia/inc/wpcom-colors-v2.php]: Not sure why these are showing up after the rebase

File: hever/inc/wpcom-editor-colors.php [hunk 2]
```diff
@@ -1,2 +1,2 @@
 <?php
-require_once get_template_directory() . '/inc/wpcom-colors-css-variables.php';
\ No newline at end of file
+require_once get_template_directory() . '/inc/wpcom-colors-css-variables.php';
```
Comment (MaggieCabrera) [inline hever/inc/wpcom-editor-colors.php]: I don't understand this one from the rebase

Comment (scruffian) [reply hever/inc/wpcom-editor-colors.php]: Files should have an empty blank line at the end.

File: rockfield/sass/_global-variables.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	--wp--preset--color--primary: #222222;
+	--wp--preset--color--primary-hover: #444444;
+	--wp--preset--color--secondary: #116821;
+	--wp--preset--color--secondary-hover: #444444;
+	--wp--preset--color--foreground: #444444;
+	--wp--preset--color--foreground-low-contrast: #757575;
+	--wp--preset--color--foreground-high-contrast: #111111;
+	--wp--preset--color--background: white;
+	--wp--preset--color--background-low-contrast: #F0F0F0;
+	--wp--preset--color--background-high-contrast: #E0E0E0;
```
Comment (scruffian) [inline rockfield/sass/_global-variables.scss:13]: Should these be the other way round? E0E0E0 is less bright than F0F0F0

**Review by scruffian — APPROVED** [review]
LGTM, just one minor comment.

File: rockfield/inc/wpcom-editor-colors.php [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<?php
+require_once get_template_directory() . '/inc/wpcom-colors-css-variables.php';
```
Comment (scruffian) [inline rockfield/inc/wpcom-editor-colors.php]: Could add a new line here...

**Comment by MaggieCabrera** [comment]
I think designers (hi @kjellr !) should have a go at this too, after finding [this issue when making the same changes to Stow](https://github.com/Automattic/themes/pull/3241#issuecomment-777435679)

File: hever/inc/wpcom-editor-colors.php (hunk 2 above)
Comment (MaggieCabrera) [reply hever/inc/wpcom-editor-colors.php]: mmh, the file that has been commited has the blank line and the one that comes from the rebase doesn't, I think that was my phpcs fixing that for me

**Pull request merged**